### DISCUSSION TITLE
No GPS metadata parsed

### DISCUSSION THREAD

**Discussion by mortbauer** [immich-app/immich#11268] [Q&A]
### The bug

The gps location isn't parsed by immich.

It is visible when I open the picture on my phone, also wenn I upload the picture via the web it is correctly there however already in the immich app on the phone the location field is empty.

Might be a permission problem.

I'm having an android phone with calyxos and granted all permissions to immich.

### The OS that Immich Server is running on

Debian Bullseye

### Version of Immich Server

v1.108.0

### Version of Immich Mobile App

v1.109.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Not related
```

### Your .env content

```Shell
Not related
```

### Reproduction steps

```bash
1. Make a picture and check if gps location set
2. in immich app location is not available and therefore not uploaded with the image
```

**Comment by mortbauer** [comment]
Ok, just verified on the phone of my girlfriend with pixel 7a and graphene that gps works just fine, since I granted all permissions as well on my phone I guess it is some strange permission thing with calyx?

**Reply by orville87** [reply]
See here: https://github.com/immich-app/immich/issues/11280

**Comment by bo0tzz** [comment]
Some phones will strip gps metadata from files when an app accesses them without having location permissions.
